helpers/_mixnes.scss

```diff
@@ -0,0 +1,12 @@
+@mixin breakpoint-max($screen,$key){
```

Comment:
no need to pass $screen as param, it's inside variables.scss
so you have access to it here